README.md

```diff
@@ -48,6 +48,7 @@ Ballerina code is distributed under [Apache license 2.0](https://github.com/ball
+||[serdes](https://github.com/ballerina-platform/module-ballerina-serdes)|[![GitHub Release](https://img.shields.io/github/v/release/ballerina-platform/module-ballerina-serdes?sort=semver&color=30c955&label=)](https://github.com/ballerina-platform/module-ballerina-serdes/releases)|[![Build](https://img.shields.io/github/workflow/status/ballerina-platform/module-ballerina-serdes/Build?label=)](https://github.com/ballerina-platform/module-ballerina-serdes/actions/workflows/build-timestamped-master.yml)|[![Trivy](https://img.shields.io/github/workflow/status/ballerina-platform/module-ballerina-serdes/Trivy?label=)](https://github.com/ballerina-platform/module-ballerina-serdes/actions/workflows/trivy-scan.yml)|[![CodeCov](https://codecov.io/gh/ballerina-platform/module-ballerina-serdes/branch/main/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-serdes)|[![Bugs](https://img.shields.io/github/issues-search/ballerina-platform/ballerina-standard-library?query=is%3Aopen%20label%3Amodule%2Fserdes%20label%3AType%2FBug&label=&color=30c955)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aopen%20label%3Amodule%2Fserdes%20label%3AType%2FBug)|[![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/ballerina-platform/module-ballerina-serdes.svg?label=)](https://github.com/ballerina-platform/module-ballerina-serdes/pulls)|[![Load Tests](https://img.shields.io/badge/-N%2FA-yellow)](https://github.com/ballerina-platform/module-ballerina-serdes/actions/workflows/process-load-test-result.yml)|
```

Comment:
This is autogenerated in ballerina-standard-library

Reply:
Noted.

Reply:
removed.